render.yaml

```diff
@@ -21,11 +21,15 @@ services:
           name: toolpad-db
           property: connectionString
       - fromGroup: toolpad-settings
+    ignoredPaths:
+     - docs/**
 
 databases:
   - name: toolpad-db
     ipAllowList: []
     plan: starter
+    ignoredPaths:
+     - docs/**
```

Comment:
I wonder if this even makes sense. I added it just in case 😅

Reply:
Actually, I'm rereviewing this PR, I think there's a problem. Shouldn't it specify the `buildFilter` property containing a `ignoredPaths` property?